### PR TITLE
Bug Fix: 'Remote' indication on job listing detail page

### PR DESCRIPTION
**Pull request by visshaljagtap**
solved #441

### CODE REVIEW

**Comment by brettski** [comment]
@visshaljagtap Thank you!
Your commit has a lint error. Something on line 124. I believe it may need to be indented.

```
08:12:08.446 [test] npm run test exited with code 0
08:12:16.136 [lint] 
08:12:16.137 [lint] /zeit/4ac276fe/next.config.original.1584450686967.js
08:12:16.137 [lint]   38:5  warning  Unexpected console statement  no-console
08:12:16.137 [lint] 
08:12:16.137 [lint] /zeit/4ac276fe/pages/partner/[slug]/job/[jobSlug].js
08:12:16.137 [lint]   124:5  error  Insert `··`  prettier/prettier
08:12:16.137 [lint] 
08:12:16.137 [lint] ✖ 2 problems (1 error, 1 warning)
08:12:16.137 [lint]   1 error and 0 warnings potentially fixable with the `--fix` option.
08:12:16.137 [lint] 
08:12:16.167 [lint] npm ERR! code ELIFECYCLE
```

File: pages/partner/[slug]/job/[jobSlug].js [hunk 1]
```diff
@@ -115,10 +115,13 @@ const AttributeTag = styled.div`
...
   background-color: ${({ color, theme }) => theme.colors[color]};
   width: fit-content;
+  border: ${({ border, theme }) =>
+    border ? `1px solid ${theme.colors.primary}` : 'none'};
 
   span {
     padding: 0.1rem 1rem;
-    color: ${({ theme }) => theme.colors.white};
+    color: ${({ fontColor, theme }) =>
+    fontColor ? theme.colors[fontColor] : theme.colors.white};
```
Comment (brettski) [inline pages/partner/[slug]/job/[jobSlug].js]: Indent two spaces

Comment (brettski) [inline pages/partner/[slug]/job/[jobSlug].js]: Lint error, line needs to be indented.

Comment (visshaljagtap) [reply pages/partner/[slug]/job/[jobSlug].js]: Yes, corrected!

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**…r/\[slug\]/job/\[jobSlug\]**](https://that-website-git-fork-visshaljagtap-fixed-remote-listing.that-conference.now.sh/partner/[slug]/job/[jobSlug]) |
|:-:|
|<a href="https://that-website-git-fork-visshaljagtap-fixed-remote-listing.that-conference.now.sh/partner/[slug]/job/[jobSlug]">Screenshot of /partner/[slug]/job/[jobSlug]</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink-cdn.s3.us-east-1.amazonaws.com/UvwaYzMUjqVBKC7EWiSTwq9UycQ_.png">(view full size)</a> |







Commit 53b0e9875bb944eb7805627fba427fa4298e042c (https://that-website-git-fork-visshaljagtap-fixed-remote-listing.that-conference.now.sh).

**Pull request merged**